packages/nc-gui-v2/composables/useSharedView.ts

```diff
@@ -0,0 +1,105 @@
+import type { ColumnType, ExportTypes, FilterType, PaginatedType, SortType, TableType, ViewType } from 'nocodb-sdk'
+import { UITypes } from 'nocodb-sdk'
+import { useNuxtApp } from '#app'
+
+const nestedFilters = ref<(FilterType & { status?: 'update' | 'delete' | 'create'; parentId?: string })[]>([])
+const paginationData = ref<PaginatedType>({ page: 1, pageSize: 25 })
```

Comment:
Why are all the refs outside the composable? If you want to share the state use `useState`.